### ISSUE TITLE
Broken red channel

### ISSUE THREAD

**Issue by HellbringerOnline** [mirsadm/motioncam#239]
Hello. I am trying to use this app on my galaxy tab a7 and i am getting broken red channel in dng files. But I have normal channels on Jpeg image. How can I fix this?

jpeg
![image](https://user-images.githubusercontent.com/14059683/163722718-012e50c8-7d2d-42b4-9ed1-5acdadd8e7ec.png)
dng
![image](https://user-images.githubusercontent.com/14059683/163722740-8fb0b4b2-a0e1-4d33-9904-3b49debaf5f0.png)
Broken Red
![image](https://user-images.githubusercontent.com/14059683/163722766-f3af29f9-7d66-425d-925a-66f4f4dfcd94.png)

**Comment by mirsadm** [comment]
Hi,
Is that when you take a photo in the app?

**Comment by HellbringerOnline** [comment]
> Is that when you take a photo in the app?
Yes, this is a preview of jpeg from the app and dgn from the app
I have another photo where this effect is less, but it is still there
![image](https://user-images.githubusercontent.com/14059683/163729071-13e4bb19-8ad7-4704-86a0-0dbe8afa7bbe.png)

**Comment by mirsadm** [comment]
hmm that is pretty weird, the red channel seems to be clipped. are you able to record a very small RAW video for me to test with?

**Comment by HellbringerOnline** [comment]
> hmm that is pretty weird, the red channel seems to be clipped. are you able to record a very small RAW video for me to test with?

Did you mean DGN frames?

**Comment by 2blackbar** [comment]
no, just container files, 1 second or something, and upload to googledrive then post link here

**Comment by HellbringerOnline** [comment]
> no, just container files, 1 second or something, and upload to googledrive then post link here

https://drive.google.com/file/d/1-fNe1NnbqdyqWnYKEiCWgK7__HsEHgzp/view?usp=sharing
